Makefile

```diff
@@ -32,10 +34,25 @@ run: build
 	docker run -p 8000:8000 $(NAME):$(VERSION)
 
 
+#
+# The 'app' target builds bandwagon Telekube package in the current directory, for example:
+#
+#   $ VERSION=1.2.0 make app
+#
+# will produce 'bandwagon-1.2.0.tar.gz'.
+#
+.PHONY: app
+app: TMPDIR := $(shell mktemp -d)
+app:
+	$(MAKE) import OPS_URL= STATE_DIR=$(TMPDIR) && \
```

Comment:
In order to clean up the temporary directory even in case any sub-commands fail, try the following:
```make
.PHONY: app
app: TMPDIR := $(shell mktemp -d)
app:
	@ret=0 && test "$$($(MAKE) build-app TMPDIR=$(TMPDIR) | tee /dev/stderr)" || ret=1;
	-rm -rf $(TMPDIR)
	@exit $$ret

.PHONY: build-app
build-app:
	$(MAKE) import OPS_URL= STATE_DIR=$(TMPDIR) && \
	$(GRAVITY) package export $(PACKAGE) $(PACKAGE_FILENAME) --state-dir=$(TMPDIR)
```